### PR TITLE
chore: remove unused langchain package

### PR DESCRIPTION
**Pull request by IrakliJani**
### Description:

these packages are not being used anywhere

### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [ ] I have reviewed the code
- [ ] I understand that "request changes" will block this PR from merging

### CODE REVIEW

**Review by magnew — APPROVED** [review]

File: packages/backend/package.json [hunk 1]
```diff
@@ -33,8 +33,6 @@
         "@aws-sdk/s3-request-presigner": "^3.272.0",
         "@casl/ability": "^5.4.3",
         "@godaddy/terminus": "^4.12.0",
-        "@langchain/core": "^0.1.51",
```
Comment (rephus) [inline packages/backend/package.json:36]: are not they used in commercial ?

Comment (rephus) [reply packages/backend/package.json:36]: also, there is a deprecation notice when you use `langhain` this is the right way to import langchain

Comment (IrakliJani) [reply packages/backend/package.json:36]: yes, but do we need them in here? we have different versions on commercial

Comment (IrakliJani) [reply packages/backend/package.json:36]: and we end up having a merge conflict every time we merge upstream

**Pull request merged**